geniza/templates/nav.html

```diff
@@ -75,23 +75,23 @@
                     </li>
                     <li class="menu-item" role="none">
                         {# Translators: label for link to 'Credits' page in main navigation #}
-                        <a href="{% slugurl "credits" %}" role="menuitem">{% translate "Credits" %}</a>
+                        <a href="/credits" role="menuitem">{% translate "Credits" %}</a>
```

Comment:
add trailing slashes please — this will work, but no need to make django serve the redirect request